tree/tree/src/TTreeCache.cxx

```diff
@@ -816,7 +818,7 @@ TBranch *TTreeCache::CalculateMissEntries(Long64_t pos, Int_t len, Bool_t all)
          // Note that we continue to iterate; fills up the rest of the entries in the cache.
       }
       // At this point, we are ready to push back a new offset
-      fMissCache->fEntries.emplace_back(std::move(iopos));
+      fMissCache->fEntries.emplace_back(iopos);
```

Comment:
Note: the `std::move()` is removed here because it has no effect (the overhead from `std::move` might even be detrimental to the performance). The IOPos type is taken by value, and it is a trivially copyable type. See also:

* https://github.com/root-project/root/blob/master/tree/tree/inc/TTreeCache.h#L87
* https://clang.llvm.org/extra/clang-tidy/checks/performance/move-const-arg.html

Reply:
Thanks for checking.